src/App.js

```diff
@@ -1,27 +1,16 @@
-import logo from './logo.svg';
 import './App.css';
+import parse from 'html-react-parser';
+import Calculator from './components/Calculator';
+import './components/styles.css';
+/* import calcWrapper from './components/Calculator';
+import calcStr from './components/Calculator'; */
+
+const calcObj = new Calculator();
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] From the project requirements, **the `App.js` should be a class-based component. Please use the materials and resources provided in order to update this function to a class-based component** 😃 👍